### DISCUSSION TITLE
Prevent the drowning of NPC's at outposts

### DISCUSSION THREAD

**Discussion by TheRealValiantic** [FakeFishGames/Barotrauma#11746] [Ideas and suggestions]
Either through an in-game setting, locking NPC's to their rooms (to also fix the heal-grab-thieving work-around. Which also seems an unintended design choice.), just making the NPC's smarter and respond/reject the player doing it,or simply giving it the same reputation hit as stealing. It has become such a meta on open servers where people think it is required to actually finish the campaign, to the point where you may as well just enable cheats through the command menu and spawn items in. There's no drawback to it, leading to an excess of loot and items, which I'd argue is an unintended effect of it. (As you don't get flagged for killing them, which is the intended way of acquiring the loot on them.) In the end, the feeling of progression is lost as it simply makes things too easy. (With the campaign following a linear curve, there is currently no way to increase the difficulty in vanilla.) 

This is likely an unpopular opinion but I wanted it said. Thank you for considering.

Edit: To clarify. Drowning NPC's through the method of putting on a suit and opening the side airlocks, while holding an NPC in place so that they die from the pressure. If someone wants to plasma-cut through the hull and flood the outpost, that'd at least be more of an intended design. (Though should still give a reputation hit.)

**Comment by BiffW** [comment]
Players can still carry radioactive materials and get npcs poisoned to death anyway. It makes no sense to make more effort to prevent players from doing the infamous outpost looting. You may slow them down, but can not stop them.

**Reply by TheRealValiantic** [reply]
A solution to that would be to make people take a reputation hit by damaging NPC's with objects/items the players carry, similar to beating a person with a crowbar, radioactive materials should take points away from reputation when used to hurt NPC's. Just because there is a plethora of ways to kill NPC's while circumventing the reputation system doesn't mean no effort should be put into fixing that. Otherwise, why is the reputation system even there to begin with? I'm not against killing NPC's, I'm against the glitching/circumventing mechanics/taking advantage of bad AI to do it without any penalties.

Ideally a server setting would be paired with that to keep both camps happy, along the lines of "Take reputation hit on damaging outpost NPC's?" but if that's not doable, I still think campaign games (public or not) will be better with less free-kill methods in it.
